pkg/handlers/business_case.go

```diff
@@ -44,14 +44,20 @@ type BusinessCaseHandler struct {
 }
 
 func requireBusinessCaseID(reqVars map[string]string, logger *zap.Logger) (uuid.UUID, error) {
+	valErr := apperrors.NewValidationError(
+		errors.New("business case failed validation"),
+		models.SystemIntake{},
+		"",
+	)
 	id := reqVars["business_case_id"]
 	if id == "" {
-		return uuid.UUID{}, errors.New("business Case ID required")
+		valErr.WithValidation("businessCaseId", "is required")
+		return uuid.UUID{}, &valErr
 	}
 	businessCaseID, err := uuid.Parse(id)
 	if err != nil {
-		logger.Error("Failed to parse business case id to uuid")
-		return uuid.UUID{}, err
+		valErr.WithValidation("businessCaseId", "must be UUID")
```

Comment:
not sure exactly how we want to delineate validation errors in the path. We use this casing for body json keys, but it's not exactly analogous to the client setting the API path with an ID.